libpod/pod.go

```diff
@@ -252,6 +252,20 @@ func (p *Pod) InfraContainerID() (string, error) {
 	return p.state.InfraContainerID, nil
 }
 
+// InfraContainer returns the infra container.
+func (p *Pod) InfraContainer() (*Container, error) {
```

Comment:
@mheon, are you cool with that?

This allows to access the infra container without having the pod's runtime object which was a nice to have in this commit.

Reply:
Sure, I have no issue with this.